config.guess

```diff
@@ -110,12 +110,12 @@ trap "rm -f \$tmpfiles 2>/dev/null; rmdir \$tmp 2>/dev/null; exit 1" 1 2 13 15 ;
  { test -n "$RANDOM" && tmp=$TMPDIR/cg$$-$RANDOM && (umask 077 && mkdir $tmp) ; } ||
  { tmp=$TMPDIR/cg-$$ && (umask 077 && mkdir $tmp) && echo "Warning: creating insecure temp directory" >&2 ; } ||
  { echo "$me: cannot create a temporary directory in $TMPDIR" >&2 ; exit 1 ; } ;
-dummy=$tmp/dummy ;
-tmpfiles="$dummy.c $dummy.o $dummy.rel $dummy" ;
+placeholder=$tmp/placeholder ;
+tmpfiles="$placeholder.c $placeholder.o $placeholder.rel $placeholder" ;
 case $CC_FOR_BUILD,$HOST_CC,$CC in
- ,,)    echo "int x;" > $dummy.c ;
+ ,,)    echo "int x;" > $placeholder.c ;
 	for c in cc gcc c89 c99 ; do
-	  if ($c -c -o $dummy.o $dummy.c) >/dev/null 2>&1 ; then
+	  if ($c -c -o $placeholder.o $placeholder.c) >/dev/null 2>&1 ; then
```

Comment:
Let's not go change upstream source code just for the sake of wording.

Reply:
Perhaps we should add (somewhere?) a note that some stuff is from upstream and that is the reason it's there?

Reply:
This is buried deep down in a file of 1530 lines. Can we please just leave this as is?

Reply:
I think the fact that the top of the file already states that it comes from gnu.org should be enough? Or we add a single note line at the top?

Reply:
I think that's enough, also:

> # Originally written by Per Bothner.  Please send patches (context
> # diff format) to <config-patches@gnu.org> and include a ChangeLog
> # entry.

clearly indicates where this comes from and how to change it.

Reply:
Based on the naming I assumed it was something created by DOMjudge, did not check the beginning so thats my mistake. Will take this out of the PR.